gemfire-core/src/main/java/com/gemstone/gemfire/distributed/internal/DistributionConfig.java

```diff
@@ -2331,4 +2354,18 @@
    */
   public void setLockMemory(boolean value);
   public boolean isLockMemoryModifiable();
+
+  /**
+   * The name of the "memory-size" property. Total memory taken by SnappyData in off-heap mode.
+   * @since SnappyData 0.9 not used in rowstore
+   */
+  public static final String MEMORY_SIZE_NAME = "memory-size";
+
+  /**
+   * The default <a
+   * href="../DistributedSystem.html#memory-size">"memory-size"</a>
+   * value of <code>""</code>.
+   * @since SnappyData 0.9 not used in rowstore
+   */
+  public static final String DEFAULT_MEMORY_SIZE = "";
```

Comment:
Once off-heap starts working well with my object changes, then this can be the default with something like 80% of available RAM like in DirectBufferAllocator.